core/amber/src/main/scala/edu/uci/ics/texera/web/storage/MongoCollectionManager.scala

```diff
@@ -0,0 +1,58 @@
+package edu.uci.ics.texera.web.storage
+
+import com.mongodb.client.model.{IndexOptions, Indexes}
+import com.mongodb.client.{FindIterable, MongoCollection, MongoCursor}
+import org.bson.Document
+
+import java.util.concurrent.TimeUnit
+import collection.JavaConverters._
+
+class MongoCollectionManager(collection: MongoCollection[Document]) {
+
+  def insertOne(document: Document): Unit = {
+    collection.insertOne(document)
+  }
+
+  def insertMany(documents: Iterable[Document]): Unit = {
+    collection.insertMany(documents.toList.asJava)
+  }
+
+  def deleteMany(condition: Document): Unit = {
+    collection.deleteMany(condition)
+  }
+
+  def getCount: Long = {
+    collection.countDocuments()
+  }
+
+  def getColumnNames: Array[String] = {
```

Comment:
Will this method be called in future PRs? I forgot why we introduced this method here.

Reply:
yes, will be called by #1640